crates/superposition_types/src/database/models/cac.rs

```diff
@@ -79,6 +79,9 @@ pub struct Dimension {
     pub position: Position,
     pub description: String,
     pub change_reason: String,
+    pub dependency_graph: Option<Value>,
+    pub immediate_parents: Option<Vec<String>>,
+    pub immediate_childrens: Option<Vec<String>>,
```

Comment:
should we rather 
```suggestion
    pub immediate_parents: Vec<String>,
    pub immediate_childrens: Vec<String>,
```
and backfill with empty array ?, instead of making it nullable, doesn't make much sense to have it wrapped it nested in Option